docs/release-notes.md

```diff
@@ -10,6 +10,73 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document gives an overview of the changes, additions, and selected fixes. Release notes published since April 2022 include Semgrep CLI, CI, and Semgrep App updates. Release notes published since April 2022 encompass all versions released during the month.
 
+## September 2022
+
+### Semgrep App
+
+#### Changes
+
+- The Findings page has been updated with UX/UI improvements to its filtering and triage functions.
+- The Dashboard page has been updated with UI improvements.
+
+#### Bug fixes
+
+- Previously, users could not receive merge request (MR) comments within GitLab repositories. This issue has been fixed. Users can now receive MR comments in GitLab from Semgrep App.
+- Update git URL parser to support optional organization after the hostname. For example `https://some.enterprise.scm/myorg/owner/repo`.
+- Various fixes and improvements to speed.
+
+### Semgrep CLI
+
+These release notes include upgrades for versions ranging between 0.112.0 and 0.115.0.
+
+#### Additions
+
+- Exclude rules by ID using CLI flag `--exclude-rule`. To exclude a specific rule, use for example `semgrep --config=auto --exclude <code><span className="placeholder">RULE_ID</span></code>. (Issue [2530](https://github.com/returntocorp/semgrep/issues/2530), PR [5974](https://github.com/returntocorp/semgrep/pull/5974))
+
+- You can now have multiple metavariables under `focus-metavariable`, which allows. Semgrep to highlight the values matched by multiple metavariables more easily in certain circumstances. For more information, see [Using multiple focus metavariables](/experiments/multiple-focus-metavariables/) documentation. (Issue [5686](https://github.com/returntocorp/semgrep/issues/5686))
+
+- You can add tags for specific projects in the Semgrep App on the configuration page of a project. To manage tags, go to the Semgrep App [Projects](https://semgrep.dev/orgs/-/projects) tab, click on the project's <i class="fa-solid fa-gear"></i> cogwheel icon to enter project configuration page, and then add or remove tags.
+    With this update, you can create `.semgrepconfig.yml` file in the root directory of your repository and add tags in this file also.
+
+    Example of tags added to `.semgrepconfig.yml` file:
+    ```yaml
+    tags:
+        - favourite
+        - awesomeproject
+    ```
+    These tags are displayed in Semgrep App [Projects](https://semgrep.dev/orgs/-/projects) tab also.
+
+    However, removing tags added through the `semgrepconfig.yml` does not remove the tag from Semgrep App. To remove these tags, remove them on the project configuration page of the Semgrep App.
```

Comment:
@zyannes updated. :-)

Reply:
Line 49 has `semgrepconfig.yml`, can we change it to `.semgrepconfig.yml`? Besides that, looks good to me!

Reply:
```suggestion
- You can add tags for specific projects in the Semgrep App on the configuration page of a project. To manage tags, go to the Semgrep App [Projects](https://semgrep.dev/orgs/-/projects) tab, click on the project's <i class="fa-solid fa-gear"></i> cogwheel icon to enter project configuration page, and then add or remove tags.
    With this update, you can create `.semgrepconfig.yml` file in the root directory of your repository and add tags in this file also.

    Example of tags added to `.semgrepconfig.yml` file:
    ```yaml
    tags:
        - favourite
        - awesomeproject
    ```
    These tags are displayed in Semgrep App [Projects](https://semgrep.dev/orgs/-/projects) tab also.

    However, removing tags added through the `.semgrepconfig.yml` does not remove the tag from Semgrep App. To remove these tags, remove them on the project configuration page of the Semgrep App.
```